lightningd/plugin.c

```diff
@@ -671,6 +672,12 @@ static bool plugin_rpcmethod_add(struct plugin *plugin,
 
 	cmd = notleak(tal(plugin, struct json_command));
 	cmd->name = json_strdup(cmd, buffer, nametok);
+	// If the command category has been specified plugin side, set it
+	if (categorytok)
+		cmd->category = atoi(buffer + categorytok->start); // category is a number
```

Comment:
Mildly suggest using a string in the JSON interface rather than a number. Possible to use a string in pylightning but that means every plugin library in every language needs to have the string->enum conversion, when we could put the string->enum conversion inside lightningd.  But no big deal.